addons/website_sale/models/website.py

```diff
@@ -118,6 +118,15 @@ def _default_recovery_mail_template(self):
         required=True,
         default='small',
     )
+    ecommerce_access = fields.Selection(
```

Comment:
Personally, I think that selection is more clear in that case, will ask Juliette.

Reply:
"I prefer to keep things as they are, I think it's clearer. The checkbox would require a longer text, which people tend not to read. IMO, we should use a checkbox when we activate something on a database. In this case, it's more of a choice that the user has to make between two options. " I tend to agree with that point of view. There is some other places where even with only two options it's still a selection field and I think in this case it is more clear to leave it as it is.

Reply:
I agree with that, but it only holds for the customer-facing field on the `res.config.settings` model; that one should indeed remain a selection. However, the field that I highlighted here is the technical one on the `website` model that users don't see; IMHO, this field should be simplified to be a boolean, and its value set based on the selection settings field. cc @paju-odoo

Reply:
I am not saying that what already is in the code for different fields is better, but it seems that even with selections with two fields its more customary to leave it as it is. Also would it simplify things since it would require to write a compute function or something similar in order to convert the values?

Reply:
It would indeed not simplify the settings' logic, but I think it's okay because it's an extra layer on the models' logic, which, I think, is the one that should be made as simple as possible. As it happens, the feature is currently broken in the latest build because you compare `self.ecommerce_access` to `'logged'` instead of `'logged_in'`, where it would be simpler and more robust to check `self.restrict_ecommerce_access`.

If you considered both options and remain convinced that it would in fact not be simpler to convert to a boolean, feel free to keep the two selections ;)

Reply:
Personally, for me the two selection is more clear and I would stick with that.